chapter09/ajd/exercise1.pl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/perl
+use utf8;
+use strict;
+use warnings;
+
+use v5.12;
+
+my %patterns = (
+  Gilligan   => qr/(?:Wiley )?Gilligan/,
+  'Mary-Ann' => qr/Mary-Ann/,
+  Ginger     => qr/Ginger/,
+  Professor  => qr/(?:The )?Professor/,
+  Skipper    => qr/Skipper/,
+  'A Howell' => qr/Mrs?. Howell/,
+);
+
+my $position = rightmost(
+  'There is Mrs. Howell, Ginger, and Gilligan',
+  \%patterns, # Change here
+);
+
+say "Rightmost match at postion $position";
```

Comment:
Nope, the Create Pull Request button sets it in stone. Something to talk about in the meeting too.